application/lib/viewmodel/menu_view_model.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import '../model/user_model.dart';
+import '../service/user_database_service.dart';
+import '../utils/route_constants.dart';
+import '../utils/text_strings.dart';
+
+class MenuViewModel with ChangeNotifier {
+  final UserDatabaseService service = UserDatabaseService();
+
+  bool _currentUserIsOrganizer = false;
+
+  bool get currentUserIsOrganizer => _currentUserIsOrganizer;
+
+  List<String> errorMessages = [];
+
+  Future<void> checkIsOrganizer(UserModel userModel) async {
+    try {
```

Comment:
Erre a függvényre valójában nincs szükség mert a userModel már tárolja, hogy organiser-ről beszélünk-e, tehát hozzáférünk úgy, hogy `userModel.isOrganizer`. Ezt a hozzáférést nem is kell kiszervezni viewModelbe, hanem mehet a ui-on.